src/index.js

```diff
@@ -81,32 +82,54 @@ class RelayCompilerWebpackPlugin {
     this.writerConfigs.default.getWriter = getWriter(options.src)
   }
 
-  apply (compiler: Compiler) {
-    compiler.plugin('before-compile', async (compilationParams, callback) => {
-      const errors = []
-      try {
-        const reporter = {
-          reportError: (area, error) => errors.push(error)
-        }
-
-        const runner = new Runner({
-          parserConfigs: this.parserConfigs,
-          writerConfigs: this.writerConfigs,
-          reporter: reporter,
-          onlyValidate: false,
-          skipPersist: true,
-        })
-
-        await runner.compileAll()
-      } catch (error) {
-        errors.push(error)
+  async compile (issuer: string, request: string) {
```

Comment:
These `issuer` and `request` paths are just placeholders at this point. The compiler will always compile all files, but that's something I'd like to improve in the future. We'll need some changes in the relay compiler itself.

`issuer` - The `FooComponent.js` path
`request` - The `__generated__/fooQuery.graphql.js` path